src/frontends/pytorch/src/utils_quantize.hpp

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+
+enum QuantizedPtNodeType { QUANTIZE_PER_TENSOR, QUANTIZE_PER_CHANNEL, DEQUANTIZE };
+
+class QuantizedPtNode : public PtFrameworkNode {
+public:
+    OPENVINO_OP("QuantizedPtNode", "util", ::ov::frontend::pytorch::PtFrameworkNode);
+    static constexpr const char* quantized_node_type_key = "QuantizedPtTypeName";
+    static constexpr const char* quantize_per_tensor_op_type_value = "aten::quantize_per_tensor";
+    static constexpr const char* quantize_per_channel_op_type_value = "aten::quantize_per_channel";
+    static constexpr const char* dequantize_op_type_value = "aten::dequantize";
```

Comment:
Probably just cosmetic, but maybe removing `aten::` part of those keys might be a good idea, given that quantize/dequantize can be called by any quantized:: op, not only those used as keys

Reply:
Agree, easier to read and shorter, applied!